config/manifests/bases/group-sync-operator.clusterserviceversion.yaml

```diff
@@ -7,15 +7,20 @@ metadata:
     categories: Security
     certified: "false"
     containerImage: quay.io/redhat-cop/group-sync-operator:latest
-    createdAt: "2021-04-28T19:33:15Z"
+    createdAt: "2021-08-01T22:11:02Z"
```

Comment:
I don't know what this `createAt` refers to. The date of the creation of the operator? The date of the latest release. If the latter can we create an action to just fix , instead of having to update it all of the times?

Reply:
The action already updates it. I just wanted to provide some more "sane" dates. not required